### ISSUE TITLE
Request failed [503]

### ISSUE THREAD

**Issue by mauritzvdworm** [cloudyr/googleCloudStorageR#142]
Started getting this error today out of nowhere.  Any idea what it means?

**Comment by MarkEdmondson1234** [comment]
Is it consistent?  It may just be a passing error due to the cloud storage service being disrupted Google's end.   If it is consistent, please add an example of what code is triggering it.

**Comment by mauritzvdworm** [comment]
Hi @MarkEdmondson1234, I have a list of csv files that I upload using gcs_upload inside an lapply.  It uploads the first couple files fine then crashes.  The crash doesn't happen with the same file being uploaded, it is pretty random when it occurs.  I'll wait a bit and try again later, maybe the issue resolves itself.

**Comment by peishenwu** [comment]
Hi everyone, here is a recent code snippet that triggers this error, I've been experiencing this error since 2021-11-11 18:11:51 CST

`library(googleCloudStorageR)`
`library(tidyverse)`
`gcs_auth(json_file = "my-service-token.json")`
`gcs_copy_object(source_object = "iris.csv", source_bucket = "my-bucket", destination_object = "iris2.csv", destination_bucket = "my-bucket")`

the output message is as follows : 
> Request failed [503]. Retrying in 1.6 seconds...
> Request failed [503]. Retrying in 1 seconds...
> Request Status Code:  503
> Trying again:  1  of  5
> Trying again:  2  of  5
> Trying again:  3  of  5
> Trying again:  4  of  5
> Trying again:  5  of  5
> All attempts failed.
> API returned: Failed to look up stub from GcsStubs

This issue seems to only affect gcs_copy_object(), any ideas ?

**Comment by MarkEdmondson1234** [comment]
The copy object URL endpoint had changed so that is now fixed.  Perhaps the other 503 is due to that too?

For regerece the www.googleapis.com changed to "https://storage.googleapis.com/"

**Comment by MarkEdmondson1234** [comment]
Any other reports for this?

**Comment by CareCT** [comment]
Hi @MarkEdmondson1234, 
I'm facing the same issue as described above. I'm using the CRAN version of `googleCloudStorageR` (v. 0.7.0) and I've noticed that both the `gcs_copy_object()` and `gcs_delete_object()` make use of the old URL ("googleapis.com/storage/..." in place of "storage.googleapis.com/..."). 

Are you planning a new CRAN release to fix this issue?

**Comment by CareCT** [comment]
Yes, the GitHub version of the package (v. 0.7.0.9000) runs smoothly for `gcs_copy_object()` and `gcs_delete_object()`.
